app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala

```diff
@@ -56,6 +56,24 @@ object Section {
 
   implicit val format: OFormat[Section] =
     OFormatWithTemplateReadFallback(SectionTemplateReads.reads)
+
+  implicit val leafExprs: LeafExpr[Section] = new LeafExpr[Section] {
+    def exprs(path: TemplatePath, t: Section): List[ExprWithPath] = t match {
+      case n: Section.NonRepeatingPage => LeafExpr(path, n.page)
+      case r: Section.RepeatingPage    => ExprWithPath(path, r.repeats) :: LeafExpr(path, r.page)
+      case a: Section.AddToList =>
+        LeafExpr(path + "title", a.title) ++
+          LeafExpr(path + "description", a.description) ++
+          LeafExpr(path + "shortName", a.shortName) ++
+          LeafExpr(path + "summaryName", a.summaryName) ++
+          LeafExpr(path + "includeIf", a.includeIf) ++
+          a.repeatsMax.toList.map(rMax => ExprWithPath(path + "repeatsMax", rMax)) ++
```

Comment:
the above line could be simplified as 

`LeafExpr(path + "repeatsMax", a.repeatsMax) ++ `

by adding to Expr object

`implicit val leafExprExpr: LeafExpr[Expr] = new LeafExpr[Expr] {
    override def exprs(path: TemplatePath, t: Expr): List[ExprWithPath] = List(ExprWithPath(path, t))
  }`